### PR TITLE
Fix Max Button for non ALPH Tokens

### PR DESCRIPTION
**Pull request by h0ngcha0**

### CODE REVIEW

File: packages/extension/src/ui/features/accountTokens/SendTokenScreen.tsx [hunk 1]
```diff
@@ -270,18 +270,19 @@ export const SendTokenScreen: FC = () => {
...
-        const balanceBn = token.balance
-
-        const maxAmount =
-          account?.networkId ===
-            "devnet" /** FIXME: workaround for localhost fee estimate with devnet 0.3.4 */
-            ? balanceBn.sub(maxFee).sub(100000000000000)
-            : balanceBn.sub(maxFee)
+      if (token.balance) {
+        let maxAmount = token.balance
+        if (feeToken?.id === token.id) {
```
Comment (polarker) [inline packages/extension/src/ui/features/accountTokens/SendTokenScreen.tsx:279]: Why don't we just check `token.id === ALPH`?

File: packages/extension/src/ui/features/accountTokens/SendTokenScreen.tsx [hunk 2]
```diff
@@ -270,18 +270,19 @@ export const SendTokenScreen: FC = () => {
...
+          maxAmount =
+            account?.networkId ===
+              "devnet" /** FIXME: workaround for localhost fee estimate with devnet 0.3.4 */
```
Comment (polarker) [inline packages/extension/src/ui/features/accountTokens/SendTokenScreen.tsx:282]: What was the issue for devnet?

Comment (h0ngcha0) [reply packages/extension/src/ui/features/accountTokens/SendTokenScreen.tsx:282]: Tested on devnet, doesn't have any issues, removing this now.

**Review by polarker — APPROVED** [review]
LGTM, but please test both ALPH and token before making the final release.

**Pull request merged**